src/test/java/org/sonarsource/sonarlint/ls/mediumtests/LanguageServerMediumTests.java

```diff
@@ -896,9 +895,19 @@ void openHotspotInBrowserShouldLogIfWorkspaceNotFound() {
     assertLogContains("Can't find workspace folder for file /workspace during attempt to open hotspot in browser.");
   }
 
+  @Test
+  void test_open_hotspot_rule_description() {
+    var FAKE_RULE_KEY = "ruleKey";
+    var FAKE_HOTSPOT_KEY = "hotspotKey";
+    var FAKE_FILE_URI = "file://sample.py";
+    var params = new SonarLintExtendedLanguageServer.ShowHotspotRuleDescriptionParams(FAKE_RULE_KEY, FAKE_HOTSPOT_KEY);
+    params.setFileUri(FAKE_FILE_URI);
+
+    assertThat(lsProxy.showHotspotRuleDescription(params)).isNull();
```

Comment:
There could be a check that the correct parameters (including the context) are passed to the client, e.g using `client.ruleDesc`.